ts/Series/Heatmap/HeatmapSeries.ts

```diff
@@ -119,7 +119,12 @@ class HeatmapSeries extends ScatterSeries {
         animation: false,
 
         /**
-         * The border radius for each heatmap item.
+         * The border radius for each heatmap itemdvxv. Border width and color
```

Comment:
Shorter and sounds better IMO:
```
The border's width and color can be set in marker options.
```

The `plotOptions.heatmap.marker.lineWidth` path is not necessary because you added reference to it in the `@see` section.